packages/suite/src/views/wallet/transactions/TradeBox/TradeBoxMenu.tsx

```diff
@@ -0,0 +1,133 @@
+import { Route } from '@suite-common/suite-types';
+import { Account } from '@suite-common/wallet-types';
+import { Button, Dropdown } from '@trezor/components';
+import { FONT_SIZE, FONT_WEIGHT } from '@trezor/components/src/config/variables';
+import { EventType, analytics } from '@trezor/suite-analytics';
+import { ReactNode } from 'react';
+import { goto } from 'src/actions/suite/routerActions';
+import { Translation } from 'src/components/suite';
+import { OverflowItems } from 'src/components/suite/OverflowItems';
+import { useDispatch } from 'src/hooks/suite';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+    & > * {
+        margin-right: 12px;
+    }
+`;
+
+const StyledDropdown = styled(Dropdown)<{ isDisabled: boolean }>`
+    background: ${({ isDisabled, theme }) => (isDisabled ? theme.BG_GREY : theme.BG_SECONDARY)};
+    width: 38px;
+    height: 38px;
+    border-radius: 8px;
+    margin-right: 0px;
+    flex-shrink: 0;
+
+    > :first-child {
+        width: 100%;
+        height: 100%;
+
+        :hover {
+            background: ${({ isDisabled, theme }) => !isDisabled && theme.BG_SECONDARY_HOVER};
+        }
+    }
+`;
+
+const StyledButton = styled(Button)`
+    font-size: ${FONT_SIZE.NORMAL};
+    font-weight: ${FONT_WEIGHT.DEMI_BOLD};
+    padding-left: 20px;
+    padding-right: 20px;
+`;
+
+const Text = styled.div`
+    position: relative;
+`;
+
+interface TradeBoxMenuProps {
+    account: Account;
+}
+
+export const TradeBoxMenu = ({ account }: TradeBoxMenuProps) => {
+    const dispatch = useDispatch();
+
+    const menuItems: {
+        removeOrder: number;
+        route: Route['name'];
+        type: 'exchange' | 'buy' | 'sell' | 'spend';
+        title: ReactNode;
+    }[] = [
+        {
+            removeOrder: 3,
+            route: 'wallet-coinmarket-exchange',
+            type: 'exchange',
+            title: <Translation id="TR_NAV_EXCHANGE" />,
+        },
+        {
+            removeOrder: 4,
+            route: 'wallet-coinmarket-buy',
+            type: 'buy',
+            title: <Translation id="TR_NAV_BUY" />,
+        },
+        {
+            removeOrder: 2,
+            route: 'wallet-coinmarket-sell',
+            type: 'sell',
+            title: <Translation id="TR_NAV_SELL" />,
+        },
+        {
+            removeOrder: 1,
+            route: 'wallet-coinmarket-spend',
+            type: 'spend',
+            title: <Translation id="TR_NAV_SPEND" />,
+        },
+    ];
+
+    return (
+        <OverflowItems
+            wrapper={Wrapper}
+            items={menuItems}
+            minVisibleItems={1}
+            visibleItemRenderer={item => (
+                <StyledButton
```

Comment:
`width: 100%` looks nice on those buttons, at least on mobile and tablet ;)

Reply:
isn't a part of the approved design

Reply:
There is no approved mobile design though, is there? :D And it's cleaner without that free space on the right, it's a minor but nice change IMO

Reply:
It is, I follow Figma design from Invity task, which is missing in Issue, I can update it with more screenshots

Reply:
I see now! Ok then 😅